applicationinsights.json

```diff
@@ -6,13 +6,19 @@
     "service.version": "${BUILD_NUMBER}"
   },
   "instrumentation": {
+    "springScheduling": {
```

Comment:
don't think this will make any difference as you're up to 3.4.10 app insights already

Reply:
will remove this bit.